modules/get-started/pages/whats-new.adoc

```diff
@@ -1,79 +1,95 @@
+You can xref:manage:security/authentication.adoc#enable-oidc[enable OIDC] for Redpanda to delegate the authentication process to an external identity provider (IdP) such as Okta, Azure AD, or on-premise Active Directory Federation Service (AD FS). When you enable OIDC, Redpanda does not need to manage user credentials directly, but can instead rely on the trusted authentication capabilities of IdPs. Redpanda's implementation of OIDC provides SASL/OAUTHBEARER support for the Kafka API, and enables standard OIDC authentication across the Schema Registry, HTTP Proxy, and Admin APIs.
 
-As data in object storage grows, the metadata for it grows. To support efficient long-term data retention, Redpanda splits the metadata in object storage, maintaining metadata of only recently-updated segments in memory or local disk, while safely archiving the remaining metadata in object storage and caching it locally on disk. Archived metadata is then loaded only when historical data is accessed. This allows Tiered Storage to handle partitions of virtually any size or retention length.
+== Audit logging 
 
-- xref:manage:tiered-storage.adoc#caching[Fine-grained caching]: To support more concurrent consumers of historical data with less local storage, Redpanda can download small chunks of remote segments to the cache directory. For example, when a client fetch request spans a subsection of a 1 GiB segment, instead of downloading the entire 1 GiB segment, Redpanda can download 16 MiB chunks that contain just enough data required to fulfill the fetch request.
+Redpanda's xref:manage:audit-logging.adoc[audit logging] supports fine-grained reporting of events in the Kafka, HTTP Proxy, Schema Registry, and Admin APIs. Events include:
 
-- xref:manage:cluster-maintenance/disk-utilization.adoc#space-management[Automatic disk space management]: Redpanda now divides disk storage into different categories to provide a flexible configuration of space: reserved disk space, cache storage, and log storage. When data usage begins to approach the target size of log storage, Redpanda does local storage housekeeping to bring usage back under the target size. This allows Redpanda to leverage available local storage safely and efficiently.
+* Changes to clusters, users, topics, ACLs, or consumer groups
+* Any time a user is authenticated
+* Any time a topic is written to or read from (requires explicit opt-in)
+* HTTP requests for the Schema Registry, HTTP Proxy, and Admin APIs
 
-- xref:manage:tiered-storage.adoc#view-space-usage[View space usage]: You can now use `rpk cluster logdirs describe` to get details about Tiered Storage space usage in both object storage and local disk.
+Auditing events can be stored in a topic, ensuring their retention for a specified period, even from Redpanda administrators. Auditing is compatible with the Open Cybersecurity Schema Framework (OCSF) and works with industry-standard tools, such as Splunk, Sumo Logic, and AWS.
```

Comment:
" ..ensuring their retention for a specified period, and are protected against removal by all users, including Redpanda administrators"